recipes/ujson/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: ujson
+  version: 1.35
+
+source:
+  git_url: https://github.com/esnme/ultrajson.git
```

Comment:
I've seen it suggested (sorry can't find a link) to use the a package instead of a git repo and tag if possible, because a git tag can be moved by a `push --force`. A link to the package (prefer `tar.gz`) can be found in [releases](https://github.com/esnme/ultrajson/releases), but the `md5` you will have to compute yourself locally because GH does not provide it on that page unfortunately.